src/prepare.js

```diff
@@ -1,28 +0,0 @@
-import path from 'node:path';
-import { existsSync, readFileSync, writeFileSync } from 'node:fs';
-import { FiltersEngine, ENGINE_VERSION } from '@cliqz/adblocker';
-
-import { BASE_PATH } from '../scripts/helpers.js';
-
-export async function generateEngine() {
```

Comment:
Is this code dead? (I'm asking because it is not obvious to me.)

Reply:
it is. npm package now uses pre-generated engine which is shipped with the package.